alembic_migration/versions/8bd0478eea2a_add_index_on_outing_date_end.py

```diff
@@ -0,0 +1,23 @@
+"""Add index on outing date_end
+
+Revision ID: 8bd0478eea2a
+Revises: 91b1beed9a1c
+Create Date: 2017-05-08 19:43:33.817581
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '8bd0478eea2a'
+down_revision = '91b1beed9a1c'
+branch_labels = None
+depends_on = None
+
+def upgrade():
+    op.create_index(
+        'outing_date_end_idx', 'outings', ['date_end'], schema='guidebook')
```

Comment:
See also https://github.com/c2corg/v6_api/issues/663#issuecomment-299944457

> I would say it should be more something like:

Similar to the feed index on date desc + id:
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/feed.py#L166-L174